tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -337,16 +339,19 @@ def setup_airgap_rancher(bastion_node, number_of_nodes=NUMBER_OF_INSTANCES):
     assert bastion_node.execute_command(
         '/snap/bin/helm')[0].find("Flags:") > -1, \
         "helm is not installed correctly"
+    
+    repo_name = "rancher-" + RANCHER_HELM_REPO
```

Comment:
Can `stable`, `latest` and `alpha` be allowed ?

Reply:
yes, this allows them all to be entered by the user.  I will have a supporting jenkins job builder PR, but you can see the same logic in the ha_deploy job.

Reply:
I see what's the goal now, to have a select dropdown with those values in the job builder.